tests/test_kci.py

```diff
@@ -1,19 +1,22 @@
 import numpy as np
 from flaky import flaky
-from sklearn.metrics.pairwise import linear_kernel, polynomial_kernel, rbf_kernel
+from sklearn.metrics.pairwise import rbf_kernel
 
 from pywhy_stats import kci
 from pywhy_stats.kernels import delta_kernel
 
 ####################################################
 # Unconditional independence tests - Continuous
 ####################################################
+seed = 12345
+np.random.seed(seed)
```

Comment:
We should not set a seed here, since will influence other downstream pytests as well. We can do it similar as in the other tests where we used a initialized `rng` object (I guess the have something like that for numpy as well?). Then, this should only be locally.

Reply:
Okay I added a RNG that is then sequentially used. Lmk if this is fine